CMakeLists.txt

```diff
@@ -67,6 +67,12 @@ IF (ENABLE_LEGACY_WEAKDEPS)
     ADD_DEFINITIONS("-DENABLE_LEGACY_WEAKDEPS=1")
 ENDIF (ENABLE_LEGACY_WEAKDEPS)
 
+# Legacy hash functions
+OPTION (WITH_LEGACY_HASHES "Build with SHA-1 and MD5 support" OFF)
+IF (WITH_LEGACY_HASHES)
+    ADD_DEFINITIONS("-DWITH_LEGACY_HASHES=1")
+ENDIF (WITH_LEGACY_HASHES)
```

Comment:
Could you add a line to `setup.py` [here](https://github.com/rpm-software-management/createrepo_c/blob/master/setup.py#L45) which turns it ON for the Python build?

Reply:
Yes, added.